apps/core/scenes/search/DatePicker.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+
+import * as React from 'react';
+import { TripInput } from '@kiwicom/margarita-components';
+import { isSameDay } from 'date-fns';
+import {
+  Icon,
+  RangeDatePicker,
+  type RangeDatePickerProps,
+  type StylePropType,
+  type IconNameType,
+} from '@kiwicom/universal-components';
+
+type Props = {|
+  +style?: StylePropType,
+  onPress: () => void,
+  label: string,
+  icon: IconNameType,
+  value: string,
+  ...$Rest<
+    RangeDatePickerProps,
+    {|
+      +onChangeTempDates: (Array<Date>) => void,
+    |},
+  >,
```

Comment:
I think it is a better approach to specify the props we are using. 
1. I can instantly see in this component what props it expects
2. The props on RangeDatePicker might change, but I don't want the props in this component to change.

Reply:
OK, done